calculator.py

```diff
@@ -1,2 +1,42 @@
 def binary_calculator(bin1, bin2, operator):
-    # your code here
+    # error checks
+    if any(c.isalpha() for c in bin1) == True:
+        return 'Error'
+    elif any(c.isalpha() for c in bin2) == True:
+        return 'Error'
+    for i in bin1:
+        if i != '1':
+            if i != '0':
```

Comment:
Instead of doing nested `if` statements like this, it's a little more readable if you do something like this:
```suggestion
        if i != '1' and i != '0':
```